__tests__/__snapshots__/sources.ts.snap

```diff
@@ -213,8 +213,8 @@ Object {
         "post_delete",
         "member_remove",
         "edit",
+        "member_role_update",
         "post_limit",
-        "moderator_add",
         "moderator_remove",
```

Comment:
Is this still needed seperate?

Reply:
Oh, right. Missed deleting this one. Thanks!